silkworm/silkrpc/core/evm_trace.cpp

```diff
@@ -1193,12 +1193,16 @@ awaitable<std::vector<Trace>> TraceCallExecutor::trace_block(const BlockWithHash
         if (!cc) {
             throw std::runtime_error("Invalid chain config");
         }
-        const auto block_rewards = protocol::EthashRuleSet::compute_reward(*cc, block_with_hash.block);
-
+        
         RewardAction action;
         action.author = block_with_hash.block.header.beneficiary;
         action.reward_type = "block";
-        action.value = block_rewards.miner;
+        action.value = 0;
+
+        if (cc->protocol_rule_set == protocol::RuleSetType::kEthash) {
```

Comment:
@elmato: Do we want to keep this behavior only when using the Ethash ruleset or do we want to only zero out when using the Trust ruleset?

Reply:
According to 
https://github.com/eosnetworkfoundation/silkworm/blob/abbfaede060e03ee03219f104a2525c09014ceb7/silkworm/silkrpc/commands/erigon_api.cpp#L268
and
https://github.com/eosnetworkfoundation/silkworm/blob/abbfaede060e03ee03219f104a2525c09014ceb7/silkworm/silkrpc/commands/ots_api.cpp#L642

It seems old silkworm code is already applying the logic of only including mining rewards for ethash

Reply:
Then I think we can stick with how things are done currently in the PR.